test/clj/rems/api/test_forms.clj

```diff
@@ -397,29 +411,26 @@
                                 :field/privacy :public}]}]
     (testing "creating"
       (testing "invalid request"
-        (letfn [(fail-request [command]
-                  (let [response (-> (request :post "/api/forms/create")
-                                     (authenticate api-key user-id)
-                                     (json-body command)
-                                     handler)]
-                    (or (= 400 (:status response))
-                        (not (get-in response [:body :success])))))]
-          (is (fail-request (assoc-in command [:form/fields 1 :field/privacy] nil)) "invalid value")
-          (is (fail-request (assoc-in command [:form/fields 1 :field/privacy] :does-not-exist)) "invalid value")
-          (is (fail-request (assoc-in command [:form/fields 0 :field/privacy] :private)) "privacy not supported")))
```

Comment:
This was never a feature of form validation, I added a test case to rems.common.form that demonstrates this